common/src/main/java/com/revenuecat/purchases/common/AppConfig.kt

```diff
@@ -16,6 +16,7 @@ class AppConfig(
     val dangerousSettings: DangerousSettings = DangerousSettings(autoSyncPurchases = true),
     // Should only be used for tests
     var forceServerErrors: Boolean = false,
+    var forceSigningErrors: Boolean = false,
```

Comment:
Maybe to be overly cautious it would help to write a unit test to verify that this is `false` by default, as a way to ensure production builds never ship with this on?

Reply:
Done in the base PR in https://github.com/RevenueCat/purchases-android/pull/1070/commits/371d747137a2ed52b723c91f385f6268f6e06635